packages/denhaag-component-library/package.json

```diff
@@ -22,13 +22,31 @@
   },
   "dependencies": {
     "@gemeente-denhaag/baseprops": "^0.2.3-alpha.50",
-    "@gemeente-denhaag/datadisplay": "^0.2.3-alpha.50",
-    "@gemeente-denhaag/input": "^0.2.3-alpha.50",
-    "@gemeente-denhaag/layout": "^0.2.3-alpha.50",
-    "@gemeente-denhaag/navigation": "^0.2.3-alpha.50",
+    "@gemeente-denhaag/alert": "^0.1.1-alpha.20",
+    "@gemeente-denhaag/avatar": "^0.2.3-alpha.50",
+    "@gemeente-denhaag/badge-counter": "^0.1.1-alpha.50",
+    "@gemeente-denhaag/divider": "^0.2.3-alpha.50",
+    "@gemeente-denhaag/list": "^0.2.3-alpha.50",
+    "@gemeente-denhaag/typography": "^0.2.3-alpha.50",
+    "@gemeente-denhaag/button": "^0.2.3-alpha.50",
+    "@gemeente-denhaag/checkbox": "^0.2.3-alpha.50",
+    "@gemeente-denhaag/formcontrol": "^0.2.3-alpha.50",
+    "@gemeente-denhaag/formcontrollabel": "^0.2.3-alpha.50",
+    "@gemeente-denhaag/formgroup": "^0.2.3-alpha.50",
+    "@gemeente-denhaag/iconbutton": "^0.2.3-alpha.50",
+    "@gemeente-denhaag/inputlabel": "^0.2.3-alpha.50",
+    "@gemeente-denhaag/radio": "^0.2.3-alpha.50",
+    "@gemeente-denhaag/select": "^0.2.3-alpha.50",
+    "@gemeente-denhaag/switch": "^0.2.3-alpha.50",
+    "@gemeente-denhaag/textfield": "^0.2.3-alpha.50",
     "@gemeente-denhaag/pickers": "^0.2.3-alpha.50",
-    "@gemeente-denhaag/stylesprovider": "^0.1.1-alpha.50",
-    "@gemeente-denhaag/surfaces": "^0.2.3-alpha.50"
+    "@gemeente-denhaag/link": "^0.2.3-alpha.50",
+    "@gemeente-denhaag/menu": "^0.2.3-alpha.50",
+    "@gemeente-denhaag/tab": "^0.2.3-alpha.50",
+    "@gemeente-denhaag/timeline": "^0.2.3-alpha.50",
+    "@gemeente-denhaag/accordion": "^0.2.3-alpha.50",
+    "@gemeente-denhaag/card": "^0.2.3-alpha.50",
+    "@gemeente-denhaag/stylesprovider": "^0.1.1-alpha.50"
```

Comment:
Can we make sure these are sorted alphabetically? npm-package-json-lint should be warning us about that with the following rule https://npmpackagejsonlint.org/docs/en/rules/dependencies/prefer-alphabetical-dependencies#example-npmpackagejsonlintrc-configuration